### PR TITLE
Add windows and macos tests to CI

### PR DESCRIPTION
**Pull request by pikinier20**

### CODE REVIEW

File: src/main/java/com/virtuslab/using_directives/custom/CommentExtractor.java [hunk 1]
```diff
@@ -50,12 +50,6 @@ private boolean skipComment() {
           while (reader.ch != CR && reader.ch != LF && reader.ch != SU) {
             reader.nextChar();
           }
-          while (reader.ch == CR) {
```
Comment (KacperFKorban) [inline src/main/java/com/virtuslab/using_directives/custom/CommentExtractor.java:53]: Why is this change needed? I would assume, that CR and LF after comments should be ignored.

Comment (pikinier20) [reply src/main/java/com/virtuslab/using_directives/custom/CommentExtractor.java:53]: With this piece of code, I had an issue that I copied line with CR but without LF (I think it's an issue that we need to keep some information to decide if we should move 1 char back or 2 chars back). Without skipping EOL the problem is gone and the only side-effect is that we've got one extra empty line at the top of file. (Which isn't a problem I think, since the trimmed file usually goes to compiler.

I think that we can stay with this solution and create some enhancement issue to not forget about that.

Comment (KacperFKorban) [reply src/main/java/com/virtuslab/using_directives/custom/CommentExtractor.java:53]: Sure, makes sense

**Review by KacperFKorban — APPROVED** [review]
